sources/packages/backend/apps/db-migrations/src/sql/ProgramYear/Create-program-year-2025-2026.sql

```diff
@@ -0,0 +1,26 @@
+INSERT INTO
+    sims.program_years(
+        program_year,
+        program_year_desc,
+        form_name,
+        is_active,
+        start_date,
+        end_date,
+        parent_form_name,
+        partner_form_name,
+        program_year_prefix,
+        max_lifetime_bc_loan_amount
+    )
+VALUES
+    (
+        '2025-2026',
+        'Study starting between August 01, 2025 and July 31, 2026',
+        'SFAA2025-26',
+        TRUE,
```

Comment:
This should be `false`:

"Please set active to false for this program year in production until advised otherwise"

Reply:
Or should we change it just for prod?